fortifyapi/client.py

```diff
@@ -25,11 +25,13 @@ def __init__(self, url: str, auth: Union[str, Tuple[str, str]], proxies=None, ve
         self.projects = Project(self._api, None, self)
         self.pools = CloudPool(self._api, None, self)
         self.workers = CloudWorker(self._api, None, self)
-        self.jobs = CloudJob(self._api, None, self)
+        self.cloudjobs = CloudJob(self._api, None, self)
+        self.jobs = Job(self._api, None, self)
```

Comment:
`self.jobs` may be too generic, would you be opposed to refactor it to `self.sscjobs`?